docker-compose.override.yml

```diff
@@ -31,9 +31,10 @@ services:
       - 1234:1234
     volumes:
       - ./playout:/src
-      - ./docker/data/playout:/app
+      - ./dev/playout:/app
+      - ./dev/certs:/certs
```

Comment:
We started packing everything also dev related in the docker directory, I felt we had to put a clear separation for what the dev files are and what the docker resources are.